### ISSUE TITLE
potato paragraph

### ISSUE THREAD

**Issue by dmil** [data4news/my-simple-website#8]
Let's improve the paragraph about potatoes. It isn't very informative. What should we add to it?

**Comment by JulieZhn** [comment]
Yes, I agree with you that it isn't informative at all, and it has some typos. We should fix the typos and add more new information about potatoes.
